crates/bevy_sprite/src/render/sprite_sheet.vert

```diff
@@ -32,6 +32,8 @@ layout(set = 2, binding = 0) uniform Transform {
 
 layout(set = 2, binding = 1) uniform TextureAtlasSprite {
     vec4 TextureAtlasSprite_color;
+    float flip_horz;
```

Comment:
wouldn't it make more sense to make these bools?

Reply:
Yes, but currently render resources doesn't support booleans.

Reply:
bawhaha oops